virtualbox-image.nix

```diff
@@ -0,0 +1,117 @@
+{ config, lib, pkgs, ... }:
+
+# This is copied from nixpkgs just with the following changes:
```

Comment:
Is there a way to 'import' and override with these changes? Is it worth it?

Reply:
Nah. It's a big blob of shell script that is the thing that is being overridden. 

Ideally, the nixpkgs one would be more configurable and we'd upstream it.